signatures/dilithium/dilithium-shake/impl/avx2.yaml

```diff
@@ -0,0 +1,10 @@
+name: AVX2, vectorized SHAKE
+platform: c
+type: optimized
+side channel guarding:
+  branching: true
+  timing: true
+dependencies:
+  - OpenSSL
```

Comment:
seems they only use it for aes, so not here, right?

Reply:
See below